graphql/resolvers.go

```diff
@@ -44,6 +44,80 @@ func (r *Resolver) Task() TaskResolver {
 
 type mutationResolver struct{ *Resolver }
 
+func (r *taskResolver) DependsOn(ctx context.Context, t *restModel.APITask) ([]*Dependency, error) {
+	depIds := []string{}
+	for _, dep := range t.DependsOn {
```

Comment:
Can we short circuit here if there are no dependencies?

Reply:
if there are no values in `t.DependsOn` then the loop will not run. Therefore is a short circuit necessary? Just seems like unnecessary code.

Reply:
The code below that will run. I think the db query will actually error if you send an empty array with $in